mockgcp/mockrecaptchaenterprise/testdata/firewallpolicy/crud/script.yaml

```diff
@@ -0,0 +1,6 @@
+- pre: gcloud config set api_endpoint_overrides/recaptchaenterprise https://public-preview-recaptchaenterprise.googleapis.com/
```

Comment:
I think this is no longer needed. Could you confirm?

Reply:
Yes good point. Will update after #4039 is merged to avoid solving merge conflict multiple times.